src/test/java/Homework_25.java

```diff
@@ -0,0 +1,25 @@
+import PageObject.LoginPage;
+
+import org.testng.annotations.Test;
+
+import static java.sql.DriverManager.getDriver;
+
+public class Homework_25 extends BaseTest {
+    @Test
+    public void loginWithValidCredentials() throws Throwable {
+        LoginPage login = new LoginPage(getDriver());
+        login.provideEmail("burul.satybaeva@testpro.io");
+        login.providePassword("Lulkerup7710%");
+        login.clickSubmit();
+        login.verifyLogin();
+    }
+
+    @Test
+    public void loginWithValidCredentials2() throws Throwable {
+        LoginPage login = new LoginPage(getDriver());
+        login.provideEmail("burul.satybaeva@testpro.io");
+        login.providePassword("Lulkerup7710%");
+        login.clickSubmit();
+        login.verifyLogin();
+    }
+}
```

Comment:
**Sample POM test (Page Object Model)**

![Clean_Test](https://github.com/Saty-lab/A53-QA-Automation-Framework/assets/104761482/903c0153-2c5b-452a-9875-b2a37eae979b)

Please, add Assert

**One test - One Assert**

Reply:
I maintain the assert inside the verifyLogin method.